ui/packages/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -97,6 +97,15 @@ define([
                 return _.extend(attributeToVal, childView.toJSON());
             }, {});
         },
+        toPropertyJSON() {
+            return {
+                properties: this.children.reduce(function(attributeToVal, childView) {
```

Comment:
❓ Any reason why we aren't returning the reduce operation directly?

Reply:
I don't know what you mean.

Reply:
Just wondering why we wouldn't return the array directly rather than in an object under the key properties.